src/styles/index.scss

```diff
@@ -1,3 +1,19 @@
-body {
+@use './header.scss';
+@use './card-container';
```

Comment:
The extension '.scss' is missing in the import statement. It should be '@use "./card-container.scss";'.